edenai_apis/apis/iointelligence/iointelligence_api.py

```
@@ -0,0 +1,104 @@
+            ProviderDataEnum.KEY, self.provider_name, api_keys=api_keys
+        )
+        self.api_key = self.api_settings["api_key"]
+        self.base_url = "https://api.intelligence.io.solutions/api/v1/"
+        self.client = OpenAI(
+            api_key=self.api_key,
+            base_url=self.base_url,
+        )
+
+    def llm__chat(
+        self,
+        messages: List = [],
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)